srcopsmetrics/entities/dependency_update.py

```diff
@@ -26,22 +26,15 @@
 
 
 class DependencyUpdate(Entity):
-    """Template entity.
+    """Python Dependency update entity.
```

Comment:
This class is not part of EnumTypes, so it cannot be used with `-e` in CLI

Reply:
not needed anymore, see this https://github.com/thoth-station/mi/pull/186#discussion_r501320326

Reply:
But if you try running with `-e` the CLI you cannot select the entities you want now

Reply:
```
pipenv run python3 srcopsmetrics/cli.py --repository thoth-station/mi -c -l -e DependencyUpdate 
Traceback (most recent call last):
  File "srcopsmetrics/cli.py", line 26, in <module>
    from srcopsmetrics.bot_knowledge import analyse_projects
  File "/home/fmurdaca/work/aicoe/mi/srcopsmetrics/bot_knowledge.py", line 27, in <module>
    from srcopsmetrics.entities import Entity, NOT_FOR_INSPECTION
ImportError: cannot import name 'Entity'
```